Upstream: Not applicable but tell them that sdjournal is not linux specific
Reason: Disable tests for features we don't have or don't enable

diff --git a/test/suite_extcaps.py b/test/suite_extcaps.py
index d086d92..a57e01d 100644
--- a/test/suite_extcaps.py
+++ b/test/suite_extcaps.py
@@ -59,10 +59,6 @@ class case_extcaps(subprocesstest.SubprocessTestCase):
         ''' extcap interface tests for androiddump '''
         check_extcap_execution("androiddump", always_present=False)
 
-    def test_ciscodump(self, check_extcap_execution):
-        ''' extcap interface tests for ciscodump '''
-        check_extcap_execution("ciscodump")
-
     def test_dpauxmon(self, check_extcap_execution):
         ''' extcap interface tests for dpauxmon '''
         if not sys.platform.startswith('linux'):
@@ -73,16 +69,6 @@ class case_extcaps(subprocesstest.SubprocessTestCase):
         ''' extcap interface tests for randpktdump '''
         check_extcap_execution("randpktdump")
 
-    def test_sdjournal(self, check_extcap_execution):
-        ''' extcap interface tests for sdjournal '''
-        if not sys.platform.startswith('linux'):
-            fixtures.skip('sdjournal is available on Linux only')
-        check_extcap_execution("sdjournal")
-
-    def test_sshdump(self, check_extcap_execution):
-        ''' extcap interface tests for sshdump '''
-        check_extcap_execution("sshdump")
-
     def test_udpdump(self, check_extcap_execution):
         ''' extcap interface tests for udpdump '''
         check_extcap_execution("udpdump")
